README.md

```diff
@@ -28,22 +27,67 @@ http://localhost:19000
 
 > login / password: **admin** / **admin**
 
-<h3 align="center">Requirements</h3>
+<h3>Collect feedback on your model</h3>
```

Comment:
Between installation and collect feedback, there is a missing section which is "Upload your model from your Python environment" 
You can reuse the same content and illustration as on the website